libs/core/src/icons/icons.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+
+const Icons = () => {
```

Comment:
please write stories in this change or next change.

Reply:
stories will be added in next change